diff --git a/qa-include/pages/login.php b/qa-include/pages/login.php
index 12946ec..7983a34 100644
--- a/qa-include/pages/login.php
+++ b/qa-include/pages/login.php
@@ -57,6 +57,7 @@ if (qa_clicked('dologin') && (strlen($inemailhandle) || strlen($inpassword))) {
 	if (qa_user_limits_remaining(QA_LIMIT_LOGINS)) {
 		require_once QA_INCLUDE_DIR . 'db/users.php';
 		require_once QA_INCLUDE_DIR . 'db/selects.php';
+		require_once QA_INCLUDE_DIR . '../qa-plugin/qa-ldap-login/qa-ldap-process.php';
 
 		if (!qa_check_form_security_code('login', qa_post_text('code'))) {
 			$pageerror = qa_lang_html('misc/form_security_again');
